.doc_gen/metadata/kms_metadata.yaml

```diff
@@ -809,6 +969,14 @@ kms_Scenario_Basics:
             - description: Define a class that wraps KMS actions.
               snippet_tags:
                 - kms.java2_actions.main   
+    PHP:
+      versions:
+        - sdk_version: 3
+          github: php/example_code/kms
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - php.example_code.kms.Basics

```

Comment:
I suggest adding in the KmsService class as well in a second excerpt (like the Java one has done). I've gotten customer feedback on other scenarios to include the service classes.

Reply:
added